src/test/java/org/jabref/logic/bibtex/comparator/BibStringDiffTest.java

```diff
@@ -31,4 +31,41 @@ void compareTest() {
         List<BibStringDiff> result = BibStringDiff.compare(originalDataBase, newDataBase);
         assertEquals(List.of(diff), result);
     }
+
+    @Test
+    void compareWithNullObject() {
+
+        assertEquals(this.diff.equals(null), false);
```

Comment:
Please make use of `assertNotEquals`:

```suggestion
        assertNotEquals(this.diff.equals, null);
```